Shared/Actions/Rich Menu/Models/RMShape.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+import AppIntents
+import SwiftUI
+
+/// Shape for an Icon's Background
+enum RMBackgroundShape: String, AppEnum {
+    static var typeDisplayRepresentation: TypeDisplayRepresentation {
+        "Background Shape"
+    }
+
+    static var caseDisplayRepresentations: [Self: DisplayRepresentation] = [
+        .circle: DisplayRepresentation(title: "Circle"),
```

Comment:
You can just use a string literal directly as the value.

Reply:
yep, the compiler was yapping about the initializer every second build and this seems to have silenced it